src/main/java/ca/bc/gov/ols/rowreader/AbstractBasicWktRowReader.java

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright © 2008-2019, Province of British Columbia
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ca.bc.gov.ols.rowreader;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.geom.Geometry;
+import org.locationtech.jts.geom.GeometryFactory;
+import org.locationtech.jts.geom.Point;
+import org.locationtech.jts.io.ParseException;
+import org.locationtech.jts.io.WKTReader;
+
+public abstract class AbstractBasicWktRowReader extends AbstractBasicRowReader {
```

Comment:
Correct me if I am wrong, but looks like some methods this class is overriding (e.g getInt, getBoolean, getDate, etc.) are not any different from the ones in AbstractBasicRowReader. Can we just drop corresponding implementations here?

Reply:
You are entirely correct. There were also a few other things in there that I cleaned up as well.